src/Logging/PsrLogger.php

```diff
@@ -241,8 +241,10 @@ public function log($level, $message, array $context = [])
 
         $entry = $this->logger->entry(
             $message,
-            $this->resource,
-            $context + ['severity' => $level]
+            $context + [
+                'severity' => $level,
+                'resource' => $this->resource
```

Comment:
Since this can be modified by a user, should we do some validation to be sure it's in a valid format?